library/src/main/java/com/pokegoapi/main/CommonRequest.java

```diff
@@ -104,4 +108,50 @@ public static GetInventoryMessage getDefaultGetInventoryMessage(PokemonGo api) {
 				CommonRequest.getDownloadSettingsMessageRequest(api));
 		return serverRequests;
 	}
+
+
+	/**
+	 * List of common requests
+	 *
+	 * @param api The current instance of PokemonGO
+	 * @return an array of ServerRequest
+	 */
+	public static ServerRequest[] commonRequests(PokemonGo api) {
+		ServerRequest[] serverRequests = new ServerRequest[4];
```

Comment:
RequestType.GET_HATCHED_EGGS is already an enum so you don´t have to add extra constants